LegacyApi/Web.config

```diff
@@ -22,7 +22,6 @@
       <add name="Pages" path="*.aspx" verb="*" type="System.Web.Handlers.TransferRequestHandler" modules="AspNetCoreModuleV2" preCondition="integratedMode,runtimeVersionv4.0" />
       <add name="ExtensionlessUrlHandler-Integrated-4.0" path="*." verb="*" type="System.Web.Handlers.TransferRequestHandler" modules="AspNetCoreModuleV2" preCondition="integratedMode,runtimeVersionv4.0" />
     </handlers>
-    <aspNetCore processPath="dotnet" arguments="..\Greenfield\bin\Debug\netcoreapp3.1\Greenfield.dll" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" hostingModel="outofprocess" />
```

Comment:
Doesn't this break running in VS?

Reply:
We can do some transform/msbuild stuff here to cause the right config to happen in the end, can't we?

Reply:
Not sure, it needs to work "in-place".

Reply:
We can always make this file development.web.config and convert it into web.config as part of the build with the right stuff on it.

Reply:
A target I introduced will transform web.config on build to point to the right place

Reply:
How does that work? Does it edit the file in-place? Does it avoid physical paths?

Reply:
We could also update it to handle any of the extensions in the static output that we want to let through

Reply:
The problem is the user might have made changes to web.config and we'll blast it away